correios/models/posting.py

```diff
@@ -380,14 +381,12 @@ class ShippingLabel:
                         "Assinatura: __________________ Documento: _______________")
     sender_header = "DESTINATÁRIO"
     carrier_logo = os.path.join(DATADIR, "carrier_logo_bw.png")
-    receiver_data_template = ("{receiver.name!s:>.40}<br/>"
-                              "{receiver.basic_address!s:>.40}<br/>"
-                              "{receiver.complement!s:>.20} {receiver.neighborhood!s:>.20}<br/>"
+    receiver_data_template = ("{receiver.name!s:>.45}<br/>"
+                              "{basic_address!s:>.95}<br/>"
```

Comment:
Seguindo sua sugestão eu criei uma property `label_address` (endereço para a shipping label) na classe `Address` ela retorna a implementação padrão (a que estava antes deste PR, mas com todas as palavras "capitalized"), enquanto as subclasses `ReceiverAddress` e `SenderAddress` quebram a linha de forma mais inteligente para cada respectivo caso.

Talvez seja o caso de explicitamente quebrar a API tornando a property `label_address` abastrata em `Address`, para forçar o uso de uma das duas e não permitir que o usuário gere pdfs de plps possivelmente bugadas.